### DISCUSSION TITLE
Custom location names are broken

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#13724] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

When using custom location names combining 

`namefile="names.txt"` in locationtypes.xml
and
`<locationnameformat.customlocation> Custom Location of [name]</locationnameformat.customlocation>` as text

The game will ignore locationnameformat and will only use names from names.txt directly

Expected (vanilla name):
![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/5a14d99f-c490-403e-8a41-6f47e71cacc6)

Result:
![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/0ae59b8e-a465-4b19-8037-f20934622b63)

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/d210a1ea-921e-40fb-bb64-0d476e866cdf)

```xml
<HuskChurch
      identifier="HuskChurch"
      replaceinradiation="Abandoned"
      namefile="%Moddir%/Text/English/HuskchurchNames.txt"
      commonnessperzone="1: 0.15, 2: 0.3, 3: 0.4, 4: 0.5"
      mincountperzone="1: 1, 2: 1, 3: 1, 4: 1, 5: 0, 6: 0"
      AllowAsBiomeGate="false" 
      Faction="huskcult" >
```

  
```
  <locationname.huskchurch>Husk Temple</locationname.huskchurch>
  <locationnameformat.huskchurch>Abbey of [name]</locationnameformat.huskchurch>
  <locationnameformat.huskchurch>Cathedral of [name]</locationnameformat.huskchurch>
  <locationnameformat.huskchurch>Chapel of [name]</locationnameformat.huskchurch>
  <locationdescription.huskchurch>A scientific outpost of the Church of Husk.</locationdescription.huskchurch>
```

[HuskchurchNames.txt](https://github.com/FakeFishGames/Barotrauma/files/14865283/HuskchurchNames.txt)

### Reproduction steps

1. Use  mod  [#DELocalHuskModIntegrated.zip](https://github.com/FakeFishGames/Barotrauma/files/14865303/DELocalHuskModIntegrated.zip)

2. Start a campaign

3. Find a random church of husk or pirate outpost

4. Name isnt using the proper format

### Version

v1.3.0.2

**Comment by Regalis11** [comment]
Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/ee5a064cff352b37fede69d05fd921c694bc418a (commit in our private repo).

I would still recommend defining the names the "new way", as entries in the xml, instead of listing them in a text file. The latter makes it impossible to translate them.

**Reply by ubertpendragon** [reply]
was confused by the fact citynames.txt still exists
